extensions/Worker.Extensions.EventGrid/src/Worker.Extensions.EventGrid.csproj

```diff
@@ -16,6 +17,12 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.16.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
We should be using System.Text, or whatever the serializer is set to with worker options; so this import should not be required

Reply:
We use JsonConvert in `EventGridBinaryDataConverter` so I think we still do need Newtonsoft.Json

Reply:
Right but we get to choose what we use, we don't have to use `JsonConvert.SerializeObject(item)` - we can use the System.Text equivalent so we don't have to bring in a new dependency. As for using the work options instead, here is an example of that:

https://github.com/Azure/azure-functions-dotnet-worker/blob/0cc8e2ac342b227cbf78fa2ffca3582bac306ac7/extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs#L164

Reply:
Removed Newtonsoft.Json dependency!